R/tags.R

```diff
@@ -0,0 +1,125 @@
+
+#' Get a simple data.frame listing all Tags on server
+#' 
+#' @param src The source object
+#' @param use_cache use the tag list previously queried 
+#' from the connect server. src$tags object is null then
+#' this parameter will be ignored
+#' 
+#' @export
+get_tags <- function(src, use_cache = FALSE){
+  validate_R6_class("Connect", src)
+  
+  res <- src$get_tags(use_cache = use_cache)
+  
+  return(res)
+}
+
+#' Get a tibble with more information for each tag
+#' 
+#' @param src The source object
+#' 
+#' @export
+get_tag_info <- function(src){
+  validate_R6_class("Connect", src)
+  
+  res <- src$get_tag_tree()
+  
+  tag_tbl <- parse_tags_tbl(res)
+  
+  return(tag_tbl)
+}
+
+#' Print a visualization of the tag tree
+#' 
+#' @param src The source object
+#' @param top_tag The value to be printed at the top of the 
+#' tag tree. This value can not have any "/" characters in it. 
```

Comment:
Do you know off-hand if Connect can have slashes in tag names? I can go check if not 😅 Do tag names w/ slashes cause trouble in this parsing approach?

Reply:
It looks like you can include slashes in your tags and it results in them being excluded from the tag tree... We can use a different separator rather than a `/` but it will mean that `dirname` and `basename` will not be able to be used to parse the tags